kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMDbusConnector.java

```diff
@@ -0,0 +1,359 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.nm.configuration;
```

Comment:
Please put this class in a different package (i.e. org.eclipse.kura.nm) since this is common for status and configuration.

Reply:
I cannot create a package with the same name of another (`org.eclipse.kura.nm`). I will move the common code inside `org.eclipse.kura.nm.common`

Reply:
Actually Eclipse is a dum-dum. I updated with the correct package.